### PR TITLE
Add config for CROWN Tutorial session

### PR DESCRIPTION
**Pull request by harrypuuter**

### CODE REVIEW

File: config/min_config.py [hunk 1]
```diff
@@ -276,9 +202,68 @@ def build_config(
...
+                scalefactors.MuonIDIso_SF,
+            ],
+            samples=["data"],
+        ),
+    )
+
+    configuration.add_shift(
+        SystematicShift(
+            name="MuonIDUp",
+            shift_config={"mm": {"muon_sf_varation": "systup"}},
```
Comment (ralfschmieder) [inline config/min_config.py:245]: muon_sf_variation and why "systup" and not just "up"? we always use "up"/"down"

File: config/min_config.py [hunk 2]
```diff
@@ -276,9 +202,68 @@ def build_config(
...
+                    scalefactors.Muon_1_ID_SF,
+                    scalefactors.Muon_2_ID_SF,
+                ]
+            },
+        )
+    )
+    configuration.add_shift(
+        SystematicShift(
+            name="MuonIDDown",
+            shift_config={"mm": {"muon_sf_varation": "systdown"}},
```
Comment (ralfschmieder) [inline config/min_config.py:257]: s.o.

File: src/pairselection.cxx [hunk 3]
```diff
@@ -1063,15 +1071,16 @@ auto ZBosonPairSelectionAlgo(const float &mindeltaR) {
  *
  * @param df the input dataframe
  * @param input_vector vector of strings containing the columns
- * needed for the alogrithm. For the muon pair selection the required paramters
- are:
+ * needed for the alogrithm. For the muon pair selection the required
+ paramters are:
```
Comment (ralfschmieder) [inline src/pairselection.cxx:1075]: parameters

**Review by ralfschmieder — APPROVED** [review]

File: config/min_config.py (hunk 1 above)
Comment (harrypuuter) [reply config/min_config.py:245]: this comes from the correction lib json this way ..

File: src/pairselection.cxx (hunk 3 above)
Comment (harrypuuter) [reply src/pairselection.cxx]: fixed

**Pull request merged**